dev/tools/localization/gen_l10n.dart

```diff
@@ -668,6 +681,23 @@ class LocalizationsGenerator {
 
     arbPathStrings.sort();
     localeInfoList.sort();
+
+    if (preferredSupportedLocales != null) {
+      for (LocaleInfo preferredLocale in preferredSupportedLocales) {
+        if (!localeInfoList.contains(preferredLocale))
```

Comment:
I personally prefer adding the parentheses for all control statements, but I don't think we require that for flutter/flutter right?

Reply:
Yes, that's right. I actually prefer adding the parentheses for control statements as well because I feel that it doesn't cause as much confusion, but got used to doing it both ways. I added it back in for this particular case